### ISSUE TITLE
Four new Wetland Potential Riverscapes Context Layers

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#122]
As we find new nationwide datasets, we should bring these in to the Riverscapes Context. This might be a good one @wally-mac for @tyler1218hatch to tackle or @philipbaileynar for @KellyMWhitehead? I'll let you guys assign. These inputs should get sucked into the National project and become avaialble for future riverscapes context projects. These specific inputs are from a new paper:
- Horvath EK, Christensen JR, Mehaffey MH, Neale AC. 2017. Building a potential wetland restoration indicator for the contiguous United States. Ecological Indicators 83 : 463–473. DOI: [10.1016/j.ecolind.2017.07.026](http://doi.org/10.1016/j.ecolind.2017.07.026) 

@shelbysawyer should be looking at these as potential optional lines of evidence in VBET, and @lauren-herbine should be looking at these as potential geoindicators for GNAT/RiverStyling.

At a minimum, I can see the following being useful to suck in:
- [ ] Compound Topographic Index - *Measure of likely wetness... a topographic index *
- [ ] poorly drained and very poorly drained soils
- [ ] potential wetland areas 
- [ ] potentially restorable wetlands on agricultural land 

Note this task minimally means, getting data in National Project, incorporating into Riverscapes Context projects workflow, creating symbology, creating business logic, testing... 

## 

> Fig. 3. Map of the spatial distribution of a) Compound Topographic Index (CTI ≥ 550), b) poorly drained and very poorly drained soils (PVP), c) combined indicator of potential wetland areas (PWA), and d) potentially restorable wetlands on agricultural land (PRW-Ag).

![Wetland Index](https://ars.els-cdn.com/content/image/1-s2.0-S1470160X17304351-gr3_lrg.jpg)

## Restorable Wetlands on Ag Land

> Fig. 4. Percent of potentially restorable wetlands on agricultural land (PRW-Ag) within 12-digit hydrologic unit (HUC) watersheds across the contiguous U.S. (CONUS).

![Ag](https://ars.els-cdn.com/content/image/1-s2.0-S1470160X17304351-gr4_lrg.jpg)

## Agreement @ Riverscape Sausage-link scale with National Wetland Inventory
![aggree](https://ars.els-cdn.com/content/image/1-s2.0-S1470160X17304351-gr2_lrg.jpg)

-----
## Data Availability
From Paper,

> Our PWA and PRW-Ag layers are currently available in the [EnviroAtlas](https://www.epa.gov/enviroatlas) (https://www.epa.gov/enviroatlas) where they can be combined with other available layers, such as land cover within riparian buffers, synthetic nitrogen fertilizer application, impaired waters, rare ecosystems, and protected lands (Fig. 6; Pickard et al., 2015). The PWA and PRW-Ag grids can also be downloaded by the user for use with other data and tools that have been designed to provide guidance on wetland placement (e.g., http://www.eli.org/freshwater-ocean/tools-evaluating-feasibility-restoration). By providing these data for download and use in a publicly available mapping application, we give decision makers the flexibility of placing potential wetlands in the context of regional or organization goals, ongoing efforts, ecosystem markets, and prioritization of further analyses.

**Comment by wally-mac** [comment]
Nice find @joewheaton! I'm excited to get this ingested into the RS context and see how useful these datasets are.  I think this is a great task for @Cashe93 with some direction from @philipbaileynar.  Cashe, I think this is an important task that you could start working on right away.  Note: Cashe talk to Philip and I about other layers that we'd like you to add to the RS context, like the 2026 National NLCD data.

**Comment by philipbaileynar** [comment]
Here are the steps. Each person who completes a task should tick them off below and then assign this ticket to the next person in the list.

1. [ ] @tyler1218hatch download the aforementioned rasters. Check they are in GeoTiff format. Leave in original projection. Put them in box and post the box links below.
1. [ ] @philipbaileynar move the files to S3.
1. [ ] @MattReimer either create a new national project for this raster collection or incorporate into existing national project.
1. [ ] @MattReimer enhance Riverscapes Context to process these rasters into the project.
1. [ ] @philipbaileynar run riverscapes context for a few projects. MHFD would be a good start.
1. [ ] @tyler1218hatch develop the business logic and symbology.
1. [ ] @philipbaileynar release new version of RAVE with aforementioned improvements.

**Comment by wally-mac** [comment]
Here is my modification of the task list based on current workloads:

- [ ] @Cashe93  download the aforementioned rasters. Check they are in GeoTiff format. Leave in original projection. Put them in box and post the box links below.
- [ ]  @philipbaileynar move the files to S3.
- [ ]  @MattReimer either create a new national project for this raster collection or incorporate into existing national project.
- [ ]  @MattReimer enhance Riverscapes Context to process these rasters into the project.
- [ ]  @philipbaileynar run riverscapes context for a few projects. MHFD would be a good start.
- [ ]  @Cashe93 with help from @tyler1218hatch develop the business logic and symbology.
- [ ]  @philipbaileynar release new version of RAVE with aforementioned improvements.

**Comment by philipbaileynar** [comment]
Thanks @wally-mac. I've removed my original list to avoid confusion. I've also assigned the issue to @Cashe93 to signify that the ball is in his court.

**Comment by joewheaton** [comment]
Thanks @philipbaileynar for the clear workflow. This will be good for us to execute. @wally-mac seemed to be suggesting @Cashe93 could do this (might be based on workload?). @wally-mac please clarify if you want @Cashe93 to do this and get up to speed on workflow (good to have multiple people who can do it). If you do assign to Cashe, just change the assignment in this ticket.

Nice one.

**Comment by wally-mac** [comment]
@joewheaton, Yep I assigned it to @Cashe93 and stated above that @tyler1218hatch is there to help train him up. FYI, Tyler is out of the office much of this month and I'm having him focus on working with Karen on the RIM tool while she's still around. Which project does this fall under? Is it on USU side? Is it on Anabranch side? Please advise. I know there's a lot of overlap but I do need to provide an index for the technicians and keep track of where hours are being spent. Thanks.

**Comment by Cashe93** [comment]
Sounds good @wally-mac, I'll get started on this.

**Comment by Cashe93** [comment]
Hey @joewheaton, I'm having a hard time locating rasters for the Compound Topographic Index and the soil drainage. Do I need to download national elevation data (NED) and SSURGO/STATSGO soil survey data so that we can produce these rasters ourselves? Or should they be available somewhere? Also, I've found the links to download the PWA and PRW-Ag rasters on the EnviroAtlas site however, the links don't appear to be working. I've sent an email to them about this and am just waiting to hear back.

**Comment by joewheaton** [comment]
Hey @Cashe93. I have only read the paper, so I do not know. DO NOT download NED and Soils data (that is a can of worms). Emailing the authors is the best you can do. 

We should all take note of just how common this is (paper has links, and hey don't work). This is what we're tyring to avoid. Also, ask @MattReimer exactly what MetaData tags and information need to be filled in so we can properly cite the source and this paper in our Riverscpaes Context projects.

Thanks.

**Comment by Cashe93** [comment]
@joewheaton, where would you like me to put these on box? I was able to download the PWA as well as the PRWag rasters (it was a browser issue). And although the CTI and Soil Drainage rasters are not included on the data download page, I was told that they might be able to send those to me. If so we should have them within the next couple of days.

**Comment by Cashe93** [comment]
@philipbaileynar [here](https://usu.box.com/s/21fqel3y7lljr7vlq7lj5iiaj1y12viq
) are the PWA and PWRag rasters. I'm still waiting to hear back on the CTI and Soil Drainage rasters.

**Comment by Cashe93** [comment]
Also, @joewheaton the full extent of the CTI is 80 GB. Is this okay, or did you only want it for a specific area?

**Comment by joewheaton** [comment]
@lauren-herbine might want to look at too.

**Comment by philipbaileynar** [comment]
@MattReimer I have put the two wetland rasters at:

s3://nardata/wetlands

These are national datasets. I suggest we make this the first standalone "thematic" riverscapes project at the national scale. i.e. let's create a separate national project just for these two rasters.

Please contact @Cashe93 if you need help understanding what these two rasters represent.
